### PR TITLE
Removes the proc loop from the BDM's AI.

### PR DESCRIPTION
**Pull request by WanderingFox95**
## About The Pull Request

Everyone's busy, the BDM is broken, I'm free, here you go. 
Ported from [https://github.com/tgstation/tgstation/commit/7c674ea486c0f8804632a96130eb0a970a4a0eb4#diff-15fc743d0b20d39ea730ce2ecc7d51171b82d2f0a1e2b13d5b002b63e7fd59a5](url)
This may also enable player-controlled BDMs to work properly, just so you know, didn't test for that though, just for y'know, the errors.

## Why It's Good For The Game

Not having a looping proc that can consume a lot of resources and possibly lag up the server (or worse) is good.

## Changelog
:cl:
fix: Removes the quick attack loop (like TG did it)
/:cl:

### CODE REVIEW

**Review by deathride58 — CHANGES_REQUESTED** [review]
needs line endings fixed

**Comment by WanderingFox95** [comment]
> 
> 
> needs line endings fixed

Not sure how they're broken.
I mean, I saved it with VCS and set it to CRLF. The file even says this when I re-open it.

**Comment by deathride58** [comment]
See the diff page; every single line is marked as changed due to line endings being swapped. Line endings need to be swapped back to what they previously were

File: code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm [hunk 1]
```diff
@@ -29,15 +23,15 @@ Difficulty: Medium
 	icon_living = "miner"
 	icon = 'icons/mob/broadMobs.dmi'
 	health_doll_icon = "miner"
-	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	mob_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
```
Comment (deathride58) [inline code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm]: mob_biotypes is a bitflag, not a list

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by WanderingFox95** [comment]
Should be fine now.

**Review by deathride58 — APPROVED** [review]
Should be testmerged (with the testmerge for disabling BDMs lifted). Aside from that, should be fine!

**Pull request merged**